lib/runner/variant-reassignment.js

```diff
@@ -5,13 +5,15 @@ import TransactionService from '../services/transaction-manager'
 
 export default class VariantReassignment {
 
-  constructor (client, logger, options = {}, retainExistingData = []) {
+  constructor (client, logger, errorCallback = this._getDefaultErrorCallback(),
```

Comment:
isnt errorCallback already part of options: https://github.com/sphereio/sphere-product-import/blob/master/lib/product-import.coffee#L23?

Reply:
Should we differ those two errorCallbacks (from reassignment and productImport)? We are passing to them different arguments.

Reply:
@butenkor It is, and we will use it. The one in here is just a default one in case you don't pass anything, e.g. for testing or when you use it with different modules.

@junajan maybe we can do it like this: you can create a special errorCallback for reassignment. If you don't, the one from productImport will be used.

Reply:
> We are passing to them different arguments.

Is it necessary to pass productDraft here? Is error object not enough (like it is in sphere-product-import)? If we throw custom error than we could extend an error with productDraft, messages or other context data. If CTP API SDK throws an error then response error object should always contain `orginalRequest` (new product or update actions) headers and response. Or you see any benefit that one can gets passed failed draft or list of actions as separated arguments into callback?

Reply:
https://github.com/commercetools/commercetools-node-variant-reassignment/pull/61/commits/542c3f2db5d4f43c20d473656d5e840cf632a2ae